Handling for generic validation error

mdrillin opened the pull request:
Addresses Issue #278 
- added handling for GenericValidationResults in CreateConnectorComponent
- some numeric inputs require positive int - added POS-INT type and sets a different min value for those on FormInputComponent
- fix issue in CreateConnectorComponent, was checking DataOptions instead of RuntimeOptions

indraraj commented on ui/packages/ui/src/app/pages/createConnector/CreateConnectorComponent.tsx:
```diff
@@ -529,6 +537,7 @@ export const CreateConnectorComponent: React.FunctionComponent<ICreateConnectorC
                 }
               }
             }
+            alert(JSON.stringify(result));
```
Need to remove the alert.

indraraj reviewed (changes requested):
UI seems to be breaking on the RuntimeOptions step for Postgres.

mdrillin commented:
Amended the commit to fix issues
- removed alert (oops)
- fixed problem in Utils which caused RuntimeStep error
- I also removed alert on DataOptions and RuntimeOptions step, since I was getting duplicate success messages as shown below on those steps:

![DuplicateMsg](https://user-images.githubusercontent.com/1820403/105064781-c4248080-5a42-11eb-90b1-0dd785c6de29.png)

indraraj commented:
Tested locally, Looks good.
